Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -68,16 +68,33 @@ module.exports = class DoubleClickToEdit {
 			this.RadioItem = Webpack.getModule(m => m?.Sizes?.NONE, { searchExports: true });
 			this.SwitchItem = Webpack.getModule(Filters.byStrings("=e.note"));
 
+			//Check for modifiers
+			this.checkForModifier = (enabled, modifier, event) => {
+				if (enabled)
+					switch (modifier) {
+						case "shift": return event.shiftKey;
+						case "ctrl": return event.ctrlKey;
+						case "alt": return event.altKey;
+					}
+				return false;
+			}
```

Comment:
> Since `checkForModifier` is used in a class you'd actually put it somewhere in the class directly. If it was a function used inside the function: `handler(e)`, you could make a local function.
> 
> The way you're doing it is by making it a class function by condition. But since there's no reason this depends on a condition, it's unnecessary. Aside from it being uncharacteristic compared to the class structure.

I'm assuming you mean within the "DoubleClickToEdit" class? Similar to "getSettingsPanel?" Or do you mean within the handler function? For example
```javascript
handler(e) {
  //Check for modifiers to edit, reply or copy
  function checkForModifier (enabled, modifier, event) {
    if (enabled)
      switch (modifier) {
        case "shift": return event.shiftKey;
        case "ctrl": return event.ctrlKey;
        case "alt": return event.altKey;
      }
    }
    return false;
  }
  
  //Handle other things
}
```

Personally I would put the function within the actual class, since I hate nested functions with a passion (they look weird), but you're the one who's going to edit this in the future, so let me know what you would like, and I'll do it :)

Reply:
Exactly like `getSettingsPanel`, yup. I've also got a small summary you can put above it if you'd like:
```
	/**
	 * 
	 * @param {boolean} enabled Is the modifier enabled
	 * @param {string} modifier Modifier key to be checked for
	 * @param {Event} event The event checked against
	 * @returns {boolean} Whether the modifier is enabled and the modifier is pressed
	 */
```

This nested function is now local (the way you showed above) or in other words: Within the scope of `handler (e)`. Which isn't bad, but would create the function each time the handler is fired; increasing load. Plugins by design are loaded as a class, thus the function won't have to be remade every time if it's directly inside the class' scope.

Having said all that; helper methods are made inside the class in this case (like `getSettingsPanel`. There aren't any nested functions anywhere else either (with the exception of the structure `getValueFromKey` 😂 ), and the `this` scope isn't lost in `handler(e)`.

This is absolutely not to bash or ridicule. And if I may, here's some tips for when editing other people's code: Have a look at how the code is structured; Are brackets after the `if()` or on the same line? Do the variables get added together by a `,` (comma) or are they explicitly typed out? Custom methods? Are they nested? Variable names? `typedOutLikeTHIS__`? The way variables are made `var`, `let`, `const`? Maybe they follow a specific code convention?

And so on. But as said previously: practice makes perfect. Don't be afraid to make mistakes, as long as you learn from them.

Reply:
Thank you for your patience! It's definitely motivating me to do more work on GitHub! And thank you for the advice, it is definitely helping! I'll be sure to pay more attention to the format of the code from now on.

I'll edit the function right now, and that should do it as far as the edits go. I'll make sure to add the summary that you included especially for further readability. I'll make sure to double-check it so that we can (hopefully) merge them soon :P